README.md

```diff
@@ -363,6 +363,12 @@ PostUp = wg set %i private-key /etc/wireguard/wg0.key <(cat /some/path/%i/privke
 
 ### Quickstart
 
+Automated Script To Install WireGuard
+
+```
+https://github.com/complexorganizations/wireguard-install
```

Comment:
There are many WireGuard install scripts, I'm not going to indorse any particular one at the top of the README so please remove this part.  You can add it to the list in the section on "tools to help with distributing keys / orchestration" though.

Reply:
Update my friend,

@pirate u wanna get some cool wireguard hoodies or shirts, maybe we can come to an arrangement..

Reply:
@pirate that was a joke but please approve this PR.